x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/three_way_diff/final_readonly/fields/description/description.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiDescriptionList } from '@elastic/eui';
+import * as ruleDetailsI18n from '../../../../translations';
+import type { RuleDescription } from '../../../../../../../../../common/api/detection_engine';
+
+interface DescriptionProps {
+  description: RuleDescription;
+}
+
+function Description({ description }: DescriptionProps) {
```

Comment:
nit: It's better to move internal functions/components down in the file so readers see public interface first.